packages/drivers/odsp-driver/src/index.ts

```diff
@@ -13,6 +13,7 @@ export * from "./odspPublicUtils";
 export * from "./odspUrlHelper";
 export * from "./createOdspUrl";
 export * from "./checkUrl";
+export * from "./createNewUtils";
```

Comment:
We should not add it. index.ts should be only for APIs that are exposed out of the driver, and this API will be used inside driver only

Reply:
ok. I thought going to be used outside driver. Thanks.